spark/src/main/scala/io/delta/tables/DeltaTable.scala

```diff
@@ -562,6 +562,165 @@ class DeltaTable private[tables](
           TableFeatureProtocolUtils.FEATURE_PROP_SUPPORTED))
     toDataset(sparkSession, alterTableCmd)
   }
+
+  /**
+   * Clone a DeltaTable to a given destination to mirror the existing table's data and metadata.
+   *
+   * Specifying properties here means that the target will override any properties with the same key
+   * in the source table with the user-defined properties.
+   *
+   * An example would be
+   * {{{
+   *  io.delta.tables.DeltaTable.clone(
+   *   "/some/path",
+   *   true,
+   *   Map("foo" -> "bar"))
+   * }}}
+   *
+   * @param target The path or table name to create the clone
+   * @param replace Whether to replace anything at the destination with the clone
+   * @param properties Any table properties to override in the clone
+   * @since 3.3.0
+   */
+  def clone(target: String, replace: Boolean, properties: Map[String, String]): DeltaTable = {
+    executeClone(table, target, replace, properties)
+  }
+
+  /**
+   * Clone a DeltaTable to a given destination to mirror the existing table's data and metadata.
+   *
+   * @param target The path or table name to create the clone
+   * @param replace Whether to replace anything at the destination with the clone
+   * @since 3.3.0
+   */
+  def clone(target: String, replace: Boolean): DeltaTable = {
+    clone(target, replace, properties = Map.empty)
+  }
+
+  /**
+   * Clone a DeltaTable to a given destination to mirror the existing table's data and metadata.
+   *
+   * @param target The path or table name to create the clone
+   *
+   * @since 3.3.0
+   */
+  def clone(target: String): DeltaTable = {
+    clone(target, replace = false)
+  }
+
+  /**
+   * Clone a DeltaTable at a specific version to a given destination to mirror the existing
+   * table's data and metadata.
+   *
+   * Specifying properties here means that the target will override any properties with the same key
```

Comment:
Could we also copy the "specifying properties...." and the "example..." to the other clone APIs as well? We adjust the example accordingly.

So for example for `cloneAtVersion(version: Long, target: String)`

`   * An example would be
   * {{{
   *  io.delta.tables.DeltaTable.cloneAtVersion(
   *   5, 
   *   "/some/path")
   * }}}`
   
   The example should be modified accordingly as well, clone for `clone` API, `cloneAtVersion` for `cloneAtVersion` APIs

Reply:
Updated